docs/basic_operation.rst

```diff
@@ -16,31 +18,31 @@ GUI-based Operation
 
 #. Pypot main window should display at this point.
 #. Click button for find potentiostat.
-#. In the drop down box the communication port that it is connected should be displayed after the program finds the correct port.
+#. In the drop down box the communication port that is connected should be displayed after the program finds the correct port.
 #. Now an experiment can be created by clicking on the add button
 #. Once clicked, a window should pop up to select the type of experiment that you want to run. Click on the appropriate experiment type.
 #. The experiment choice window should close and a new window should appear that allows you to edit the experiment parameters.
-#. Once experiment parameters input, click generate preview to show the time vs. voltage profile to verify that your experimental procedure appears correct.
+#. Once experiment parameters are input, click generate preview to show the time vs. voltage profile to verify that your experimental procedure appears correct.
 #. Once satisfied, click save experiment file and then save the file as a .yml file.
 #. Now you can begin the experiment by pressing the start button beside the start/stop experiment label.
-#. Once experiment is complete, click disconnect the disconnect potentiostat button.
+#. Once experiment is complete, click the disconnect potentiostat button.
```

Comment:
We should add a comment here to mention what to do if the user wants to run another experiment. There could be a couple cases. First they could want to change settings (which they shouldn't have to shut down for). Second, they could want to switch electrodes and run the same experiment. Would they need to disconnect and shut down before touching the clips, or can they make changes to the cell at this stage and just click rerun?

Reply:
Well for this one, I was under the guise that it was just for basic operation and we wouldn't explore all the possibility space (this could get ridiculously long for the GUI case). So I was just thinking we showed how to run a single trial basic. They can touch clips after experiment without disconnecting.

Reply:
Ok, that works. This should be fine since the clips are safe to touch at this stage (just in case people are curious/adventurous and decide to try more things).